pages/sobre-nos.vue

```diff
@@ -1,5 +1,174 @@
 <template>
-  <div>
-    <h2>Sobre nós</h2>
+  <div class="about-us__container">
+    <div class="about-us__container-content">
+      <div class="about-us__container-content-emtu">
+        <h2 class="about-us__container-content-title">
+          Sobre o <span>projeto</span>
+        </h2>
+        <p class="about-us__container-content-paragraph">
+          O software proposto faz o mapeamento entre as classificações dos níveis de
+          deficiência dos passageiros e as adequações necessárias para que esses passageiros
+          utilizem o sistema de transporte público metropolitano da melhor forma possível.
+          Os dados e informações a serem utilizados neste projeto serão da Região Metropolitana do Vale do
+          Paraíba e Litoral Norte de São Paulo (RMVPLN) e da Empresa Metropolitana de Transportes Urbanos de São Paulo (EMTU),
+          parceira deste projeto de extensão.
+        </p>
+        <p class="about-us__container-content-paragraph">
+          O software  relaciona os CID dos passageiros PcD com suas necessidades
```

Comment:
🚲 só pra explicar o que é essa sigla
```suggestion
          O software  relaciona os CID (Código Internacional de Doenças) dos passageiros PcD com suas necessidades
```